cmd/security-secretstore-setup/res/configuration.toml

```diff
@@ -23,20 +23,22 @@ EnableRemote = false
 File = './logs/security-secretstore-setup.log'
 
 [SecretService]
-scheme = "https"
-server = "edgex-vault"
-port = 8200
-certpath = "v1/secret/edgex/edgex-security-proxy-setup/kong-tls"
-cafilepath = "/vault/config/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem"
-certfilepath = "/vault/config/pki/EdgeXFoundryCA/edgex-kong.pem"
-keyfilepath = "/vault/config/pki/EdgeXFoundryCA/edgex-kong.priv.key"
-tokenfolderpath = "/vault/config/assets"
-tokenfile = "resp-init.json"
-vaultsecretshares = 5
-vaultsecretthreshold = 3
-tokenprovider = ""
-tokenproviderargs = [ "-confdir", "res/token-file-provider" ]
-tokenprovidertype = "oneshot"
+Scheme = "https"
+Server = "edgex-vault"
```

Comment:
I'm wondering if this even worked before this change in the non-docker configuration...  Usually this is "localhost".  Suggest change to "localhost" if this is indeed broken.

Reply:
The reason we have configuration.toml file outside the res folder is to make snap project easier to covert to whatever formats/values that work in the snap environment, and we just keep them identical first, then another PR will be followed to address snap issues - we don't have such environment to test/verify.